oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -61,8 +62,8 @@ because it is no longer in use, the following steps should be followed.
 
 Transfer to New Owner if Interest
 ---------------------------------
-First, if the repository is public, follow these steps to see if anyone would like
-to take ownership of it:
+First, if the repository is public, and a part of openedx releases follow these
+steps to see if anyone would like to take ownership of it:
```

Comment:
Nit: update comma to the end of the `if` clause:
`if the repository is public and a part of openedx releases,`